recipe detail swapper customization (#24)
miamtech/mealz-documentation: docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailSwapper.mdx

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ### swapper

Reviewer: ```suggestion
### Swapper
```

> To create your own recipe detail swapper template you have to create class that implement `Swapper`

Reviewer: ```suggestion
To create your own recipe detail swapper template you create a class that implements `Swapper`
```

            ) {
                Text(
                    text = Localisation.recipeDetails.shopping.localised,
                    style = TextStyle(
                        color = if (isShopping) ai.mealz.sdk.theme.Colors.boldText else ai.mealz.sdk.theme.Colors.white,
                        fontSize = 16.sp,
                        textAlign = TextAlign.Center,
                        fontWeight = FontWeight.Bold
                    ),
                    modifier = Modifier
                        .weight(1f)

                )
                Text(
                    text = Localisation.recipeDetails.cooking.localised,
                    style = TextStyle(
                        color = if (isShopping) ai.mealz.sdk.theme.Colors.white else ai.mealz.sdk.theme.Colors.boldText,
                        fontSize = 16.sp,
                        textAlign = TextAlign.Center,
                        fontWeight = FontWeight.Bold
                    ),
                    modifier = Modifier.weight(1f)
                )
            }
        }
    }
}
```
</TabItem>
</Tabs>

with 

```kotlin
data class SwapperParameters(

>     val isShopping: Boolean, // true if we are is shooping mode (you can add product to basket)

Reviewer: ```suggestion
    val isShopping: Boolean, // true if we are is shopping mode (you can add product to basket)
```
